ci/lint.yml

```
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: onsdigital/dp-concourse-tools-lint-go
-    tag: latest
+    tag: 1.24.4-bullseye-golangci-lint-2
```

This does not need to be pinned. Also `latest` is an alias for `1.24.4-bullseye-golangci-lint-2` anyway.
```suggestion
    tag: latest
```